test/config-next/ocsp-responder.json

```diff
@@ -4,6 +4,19 @@
       "dbConnectFile": "test/secrets/ocsp_responder_dburl",
       "maxOpenConns": 10
     },
+    "redis": {
+      "username": "ocsp-responder",
+      "passwordFile": "test/secrets/ocsp_responder_redis_password",
+      "addrs": [
+        "10.33.33.7:4218"
```

Comment:
How often with rocsp attempt to redial these addresses? Let's say a redis cluster is running, we perform a change to the job spec and the redis cluster is redeployed on different ports, will the ocsp-responder be aware of that or will SRE need to manually restart the ocsp-responder?

Reply:
Nope, intended architecture is:

```
[relying party] --> [ocsp-responder] --> [redis cluster]
```

I think most likely in order to support live redeploys like that, we'll need to support lookup of backends via SRV records (which would be a separate change from this one). Would that work for you?

Reply:
Yes it would.

Reply:
I know I gave this feedback earlier in a meeting, but:

1. Redis node ports will now be statically assigned so we won't have to worry about cluster members changing their ports each time the allocation is updated
2. If even a single healthy Redis Cluster member is reachable then `go-redis` will be able to learn about all new cluster members. Cluster members gossip about one another regularly for heartbeats, but they also gossip about every `MEET` interaction. When I add a new node to an existing cluster I need only introduce that node to one existing member, the rest will learn about that node via gossip.

Reply:
I think all of this thread is informational. Thanks for the discussion! I don't think there is any todo item for me here, and it looks like questions are answered so I am going to resolve. Please reopen and let me know if there is a change you would like to see from this thread.